aries-site/src/examples/templates/forms/do-dont/ColumnFormDont.js

```diff
@@ -0,0 +1,299 @@
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  // provide order of formfields for validation
+  // to properly place focus on any errors or infos
+  const formFields = [
+    'firstName',
+    'lastName',
+    'address1',
+    'address2',
+    'city',
+    'state',
+    'zipcode',
+    'isBusiness',
+    'fullName-shipping',
+    'phone-shipping',
+    'email-shipping',
+  ];
+
+  // On long forms, we want to focus the first of any fields that
+  // return an error or info message. This removes the need for the
+  // user to scroll to find which field blocked form submission.
+  const onValidate = validationResults => {
+    const target = formFields.find(
+      field =>
+        field in validationResults.errors || field in validationResults.infos,
+    );
+    if (target) {
+      const targetFormField = document.getElementsByName(target);
+      targetFormField[0].focus();
+    }
+  };
+
+  return (
+    <Box gap="medium">
+      <Header
+        direction="column"
+        align="start"
+        gap="xxsmall"
+        pad={{ horizontal: 'xxsmall' }}
+      >
+        {/* Use semantically correct heading level and adjust size as 
+        needed. In this instance, this example is presented within an 
+        HTML section element and this is the first heading within the 
+        section, therefor h2 is the semantically correct heading. For 
+        additional detail, see https://design-system.hpe.design/foundation/typography#semantic-usage-of-heading-levels). */}
+        <Heading level={2} margin="none">
+          Shipping
+        </Heading>
+        <Text>for your HPE products</Text>
+      </Header>
+      <Box
+        // Padding used to prevent focus from being cutoff
+        pad={{ horizontal: 'xxsmall' }}
+      >
+        <Form
+          value={formValues}
+          onChange={setFormValues}
+          messages={{
+            required: 'This is a required field.',
+          }}
+          onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          onValidate={onValidate}
+          method="post"
+          validate="submit"
+        >
+          <Box gap="large" direction="row">
+            <Box>
+              <Text size="large" margin={{ bottom: 'xsmall', top: 'none' }}>
+                Shipping Information
+              </Text>
+              <Text
+                margin={{ horizontal: 'none', vertical: 'xsmall' }}
+                size="xsmall"
+              >
+                Shipping Address *
+              </Text>
+              <FormField
+                required
+                htmlFor="firstNameShipping"
+                name="firstNameShipping"
+                label="First Name"
+                contentProps={{ width: 'medium' }}
+              >
+                <TextInput id="firstNameShipping" name="firstNameShipping" />
+              </FormField>
+              <FormField
+                required
+                htmlFor="lastName"
```

Comment:
We have to adjust the htmlFor, id, name, etc for all of the inputs in this example so they're unique. Then, we will also have to update the "formFields" on line 94 to match

Reply:
I took out the first 2 per Matts suggestion below I didn't realize we are asking for firstname lastname then full name